invenio_jsonschemas/config.py

```diff
@@ -44,3 +44,25 @@
 
 JSONSCHEMAS_REGISTER_ENDPOINTS_UI = True
 """Register the endpoints on the UI app."""
+
+JSONSCHEMAS_SCHEMAS = None  # loads all json schemas
+"""List of entry names to register json schemas for.
+
+If `None`, all json schemas defined through the ``invenio_jsonschemas.schemas``
+entry point in setup.py will be registered.
+If `[]`, no json schemas will be registered.
```

Comment:
```suggestion
If ``[]``, no json schemas will be registered.
```

I think you need double-backticks for Shinx docs to display them as code blocks (but I might be wrong). Check other places as well

Reply:
I checked `invenio-search` and it's using double-backticks as well so I changed it.